### DISCUSSION TITLE
질문2

### DISCUSSION THREAD

**Discussion by maxking2007** [storycraft/node-kakao#438] [Q&A]
4버전에서 답장으로 답장된 메시지를 가리거나 유저를 강퇴하는것은 어떻게해야하나요? 예제를 간단하게라도 써주신다면 감사하겠습니다

**Comment by generalxd** [comment]
코드를 한번이라도 보셨거나
Discussion에 검색 해보셨나요 ?

**Reply by maxking2007** [reply]
전에 올라왔던글은 3버전이라서 4버전에는 맞지않았고 examples에 소스들을 여러개 보고 공부했지만 답이 나오지않아 질문한것입니다

**Reply by generalxd** [reply]
channelList.open 객체에 관련 함수들이 있습니다.

**Comment by raravel** [comment]
https://github.com/storycraft/node-kakao/blob/967d0355f45b108a282d94d20547e37b2670b161/src/talk/openlink/talk-open-channel.ts#L299-L301
